vars/FreeBSD.yml

```diff
@@ -1,3 +1,11 @@
+__argus_radium_user: argus
+__argus_radium_user_home: /var/log/argus
+__argus_radium_user_shell: /usr/sbin/nologin
+__argus_radium_user_uid: 561
```

Comment:
FreeBSD's `argus` and `argus-clients` packages are broken in many ways. it is clearly obvious that they are not maintained because the packages do not work _by default_ (see SASL workarounds). the lack of rc.subr script for `radium`, UID/GID is not registered, and DEBUG option, which is almost always required when you run `argus`, is not available. a simple PR does not fix the issues. the ports need to be overhauled.

you may create a user and a group before the rpm messes. but Red Hat variants do not manage UID/GID, which causes collision.

I am creating ID-less version of the branch, which would clarify the point.

the way BSD projects manage UIDs/GIDs is deemed to break. registered user names are prone to collide, and the numbers will eventually exhaust. OpenBSD does not have the former issue, though.

I once thought that strictly managing IDs is a good idea, but now realize that it is not because, if you have multiple distributions, it is editable that some IDs  would collide anyway. also, I have not bumped into issues caused by un-managed IDs for years.